pkg/chartverifier/reportBuilder.go

```diff
@@ -150,3 +162,43 @@ func GenerateSha(rawFiles []*helmchart.File) string {
 
 	return fmt.Sprintf("sha256:%x", chartSha.Sum(nil))
 }
+
+func GetPackageDigest(uri string) string {
+
+	url, err := url.Parse(uri)
+	if err != nil {
+		return ""
+	}
+	var chartReader io.Reader
+	switch url.Scheme {
+	case "http", "https":
+		var chartGetResponse *http.Response
+		chartGetResponse, err = http.Get(url.String())
+		if err == nil {
+			chartReader = chartGetResponse.Body
```

Comment:
Lets say I expect url to be valid and this still fails. Is there a way to see this error? Also do you happen to know if http.Get() support redirects? Have you tested with IBM raw paths?

Reply:
This is one weakness of the cobra command structure we only output a message if the command fails and no report results. One suggestion I have is to add a messages attribute to the tool metadata where we include such errors.

Reply:
Like a verbose run? We can do that in